File: Cristian_viana_camacho/programas/ruby/juego_de_adivinanzas.rb
puts "Binevenid@!!"
puts "selecciona una adivinanza!!"
puts "|  ad1  |  ad2  |  ad3  |"
puts "los mas dificiles :D"
puts "|  ad4  |  ad5  |  ad6  |"

nda = gets.chomp

if nda == "ad1"
    puts "PISTA: es un Animal"
    puts "Vuelo de noche, duermo en el día y nunca veras plumas en ala mía"
    r1 = gets.chomp
    if r1 == "murcielago"
        puts"su respuesta es..."
        puts"CORRECTA!"
    elsif r1 == "Murcielago"
        puts"su respuesta es..."
        puts"CORRECTA!"
    else 
        puts"su respuesta es..."
        puts"INCORRECTA!"
    end
elsif nda == "ad2"
    puts "PISTA: es un Animal"
    puts "¿Cuál es la estrella que no tiene luz?"
    r1 = gets.chomp
    if r1 == "la estrella de mar"
        puts"su respuesta es..."
        puts"CORRECTA!"
    elsif r1 == "estrella de mar"
        puts"su respuesta es..."
        puts"CORRECTA!"
    else 
        puts"su respuesta es..."
        puts"INCORRECTA!"
    end


elsif nda == "ad3"
    puts "PISTA: es una fruta"
    puts "Blanca por dentro, verde por fuera. Si quieres te lo digo, espera..."
    r1 = gets.chomp
    if r1 == "La pera"
        puts"su respuesta es..."
        puts"CORRECTA!"
    elsif r1 == "la pera"
        puts"su respuesta es..."
        puts"CORRECTA!"
    else 
        puts"su respuesta es..."
        puts"INCORRECTA!"
    end


elsif nda == "ad4"
    puts "Una señorita muy señoreada, lleva sombrero verde y blusa colorada."
    r1 = gets.chomp
    if r1 == "La fresa"
        puts"su respuesta es..."
        puts"CORRECTA!"
    elsif r1 == "la fresa"
        puts"su respuesta es..."
        puts"CORRECTA!"
    else 
        puts"su respuesta es..."
        puts"INCORRECTA!"
    end


elsif nda == "ad5"
    puts "¿Qué cosa es que cuanto más le quitas más grande es?"
    r1 = gets.chomp
    if r1 == "un agujero"
        puts"su respuesta es..."
        puts"CORRECTA!"
    elsif r1 == "un hueco"
        puts"su respuesta es..."
        puts"CORRECTA!"
    else 
        puts"su respuesta es..."
        puts"INCORRECTA!"
    end


elsif nda == "ad6"
    puts "¿Qué es lo que se hace de noche, que no se puede hacer de día?"
    r1 = gets.chomp
    if r1 == "trasnocharse"
        puts"su respuesta es..."
        puts"CORRECTA!"
    elsif r1 == "Trasnocharse"
        puts"su respuesta es..."
        puts"CORRECTA!"
    else 
        puts"su respuesta es..."
        puts"INCORRECTA!"
    end
end

File: Cristian_viana_camacho/programas/dart/juego_de_adivinanzas.dart
import'dart:io';

void main() {
  print('Binevenid@!!');
  print('selecciona una adivinanza!!');
  print('|  ad1  |  ad2  |  ad3  |');
  print('los mas dificiles :D');
  print('|  ad4  |  ad5  |  ad6  |');
  String nda = stdin.readLineSync()!;

  if(nda == 'ad1'){
    print('PISTA: es un Animal');
    print('Vuelo de noche, duermo en el día y nunca veras plumas en ala mía');
    String r1 = stdin.readLineSync()!;
    if(r1 == 'murcielago'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else if (r1 == 'Murcielago'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else{
      print('su respuesta es...');
      print('INCORRECTA!');
    };


  }else if(nda == 'ad2'){
    print('PISTA: es un Animal');
    print('¿Cuál es la estrella que no tiene luz?');
    String r1 = stdin.readLineSync()!;
    if(r1 == 'estrella de mar'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else if (r1 == 'la estrella de mar'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else{
      print('su respuesta es...');
      print('INCORRECTA!');
    };


  }else if(nda == 'ad3'){
    print('PISTA: es una fruta');
    print('Blanca por dentro, verde por fuera. Si quieres te lo digo, espera...');
    String r1 = stdin.readLineSync()!;
    if(r1 == 'la pera'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else if (r1 == 'La pera'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else{
      print('su respuesta es...');
      print('INCORRECTA!');
    };


  }else if(nda == 'ad4'){
    print('Una señorita muy señoreada, lleva sombrero verde y blusa colorada.');
    String r1 = stdin.readLineSync()!;
    if(r1 == 'la fresa'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else if (r1 == 'La fresa'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else{
      print('su respuesta es...');
      print('INCORRECTA!');
    };
  

  }else if(nda == 'ad5'){
    print('¿Qué cosa es que cuanto más le quitas más grande es?');
    String r1 = stdin.readLineSync()!;
    if(r1 == 'un agujero'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else if (r1 == 'un hueco'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else{
      print('su respuesta es...');
      print('INCORRECTA!');
    };

  
  }else if(nda == 'ad6'){
    print('¿Qué es lo que se hace de noche, que no se puede hacer de día?');
    String r1 = stdin.readLineSync()!;
    if(r1 == 'trasnochar'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else if (r1 == 'Trasnochar'){
      print('su respuesta es...');
      print('CORRECTA!');
    }else{
      print('su respuesta es...');
      print('INCORRECTA!');
    };
  }

}
